src/Sarif.FunctionalTests/SpecExamples/RuleMetadata.sarif

```diff
@@ -3,15 +3,25 @@
   "runs": [
     {
       "tool": {
-        "name": "CodeScanner",
+        "name": "BinaryAnalyzer",
         "semanticVersion": "2.1.0"
       },
       "rules": {
-        "C2001": {
-          "id": "C2001",
-          "fullDescription": "A variable was used without being initialized. This can result in runtime errors such as null reference exceptions"
+        "BA2006": {
+          "id": "BA2006",
+          "name": "BuildWithSecureTools",
+          "shortDescription": "Application code should be compiled with the most up-to-date tool sets ...",
+          "fullDescription": "Application code should be compiled with the most up-to-date tool sets ...",
+          "messageFormats": {
+            "Error_BadModule": "built with {0} compiler version {1} (Front end version {2})",
+            "Pass": "{0}' was built with tools that satisfy configured policy.",
+            "Error": "{0}' was compiled with one or tools that do not satisfy configured policy ...",
```

Comment:
Again, since this text looks fine (complete sentence) why provide apparent continuation markers?